coresimd/src/aarch64/neon.rs

```diff
@@ -39,41 +41,43 @@ pub unsafe fn vaddd_u64(a: u64, b: u64) -> u64 {
 
 #[cfg(test)]
 mod tests {
-    use super::*;
+    use super::f64x2;
+    use aarch64::neon;
+    use stdsimd_test::simd_test;
 
-    #[test]
-    fn vadd_f64_() {
+    #[simd_test = "neon"]
```

Comment:
The `simd_test` macro now works for `arm`-like targets and it should also work for powerpc and mips as well.